src/app/auth/login/page.tsx

```diff
@@ -1,75 +1,89 @@
-'use client'
+'use client';
 import login from '@/app/api/auth/login';
 import { Button, Checkbox, Form, Input } from 'antd';
 import Swal from 'sweetalert2';
+import Image from 'next/image'
+import './main.css';
 
 export default function Page() {
-    const onFinish = ( values: any ) => {
+    const onFinish = async ( values: any ) => {
```

Comment:
Same here, and you can destructure the values variable if you set the type for login.